src/routes/docs/products/databases/bulk-operations/+page.markdoc

```diff
@@ -0,0 +1,150 @@
+---
+layout: article
+title: Bulk Operations
+description: Perform bulk operations on documents within your collections for efficient data handling.
+---
+
+Appwrite Databases supports bulk operations for documents, allowing you to create, update, or delete multiple documents in a single request. This can significantly improve performance for apps as it allows you to reduce the number of API calls needed while working with large data sets.
+
+As of now, bulk operations can only be performed via the server-side SDKs. The client-side SDKs do not support bulk operations.
+
+# Create documents {% #create-documents %}
+
+{% info title="Permissions required" %}
+You must grant **create** permissions to users at the **collection level** before users can create documents.
+[Learn more about permissions](/docs/products/databases/permissions)
+{% /info %}
+
+You can create multiple documents in a single request using the `createDocuments` method.
+
+{% multicode %}
```

Comment:
do we need this to be `multicode`? Its only one example, right? same goes for others too.